lib/mongoid/persistable/datable.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Mongoid
+  module Persistable
+
+    # Defines behavior for setting a field (or fields) to the current
+    # date/time.
+    module Datable
+      extend ActiveSupport::Concern
+
+      # Converts the argument into the appropriate $currentDate type.
+      #
+      # @param [ nil | String | Symbol | Hash ] type The type description
+      #   to translate.
+      #
+      # @return [ true | Hash ] The type description indicated by the argument.
+      #
+      # @api private
+      def self.translate_date_field_spec(type)
+        case type
+        # `current_date(:foo)` == `current_date(foo: true)`
+        when nil then true
+        when "timestamp", :timestamp, "date", :date
```

Comment:
Consider allowing :time as an alias for timestamp here.

Reply:
> how does MongoDB know the timezone in order to determine the date?

Excellent point. The answer is "it doesn't". It will use (I believe) whatever time and date are configured on the server. In a world where time zones matter, it seems more useful to explicitly set the timestamp to an appropriate value, rather than letting the server do it.

I'll discuss with the team; I still think it's important to have functions from the underlying API accessible, but maybe this one could use *less* visibility.